src/jobs/config_sync_job.ts

```diff
@@ -201,9 +203,19 @@ export class ConfigSyncJob extends SyncJob {
         // scenario c)
         operationsOk &&= await this._deleteMapping(mapping);
       }
-
+      //TODO figure out how to pass TogglTrack in better
+      const togglService = secondaryServicesWrappersMap.get("TogglTrack");
       // console.log('[OMR] Archiving '.concat(timeEntriesToArchive.length.toString(), ' TESOs for user ', this._user.username,'.'));
       for (const timeEntryToArchive of timeEntriesToArchive) {
+
+        if (togglService !== undefined) {
+          let toggleTimeEntry = timeEntryToArchive.serviceTimeEntryObjects.find(
+              (element) => element.service === "TogglTrack");
+          if (toggleTimeEntry !== undefined && timeEntryToArchive.issueName !== undefined) {
+            await togglService.syncedService.replaceTimeEntryDescription(toggleTimeEntry, timeEntryToArchive.issueName)
+          }
```

Comment:
prislo by mi vhodne obalit do try-catch bloku, aby sme si boli 100% isti, ze sa prevedie nasledovna archivacia TESO na riadku 219.

Reply:
yes,dava mi zmysel nastavit operationsOk na false.
Zaroven by sme tam mohli lognut infor hlasku na Sentry.

Reply:
OperationsOk jsem nastavil na false. Hlášku bych v tomto místě nepřidával. Nejedná se o critical failure, ale nice to have věc při archivaci tagu.